go.mod

```diff
@@ -3,18 +3,20 @@ module github.com/cosmos/gaia/v5
 go 1.16
 
 require (
-	github.com/cosmos/cosmos-sdk v0.42.6
+	github.com/cosmos/cosmos-sdk v0.42.9
 	github.com/gorilla/mux v1.8.0
 	github.com/gravity-devs/liquidity v1.2.9
 	github.com/pkg/errors v0.9.1
 	github.com/rakyll/statik v0.1.7
 	github.com/spf13/cast v1.3.1
 	github.com/spf13/cobra v1.1.3
 	github.com/stretchr/testify v1.7.0
-	github.com/tendermint/tendermint v0.34.11
+	github.com/tendermint/tendermint v0.34.13
 	github.com/tendermint/tm-db v0.6.4
 )
 
 replace google.golang.org/grpc => google.golang.org/grpc v1.33.2
 
+replace github.com/tendermint/tendermint => github.com/tendermint/tendermint v0.34.13
```

Comment:
this line probably redundant if you are bumping versions above

Reply:
It is overwriting the SDK version

Reply:
yup this is to force the SDK to use that version of tendermint too